plugin/evm/vm_upgrade_bytes_test.go

```diff
@@ -157,76 +157,10 @@ func TestVMUpgradeBytesPrecompile(t *testing.T) {
 	assert.Equal(t, signedTx1.Hash(), txs[0].Hash())
 }
 
-func TestVMUpgradeBytesNetworkUpgrades(t *testing.T) {
```

Comment:
Can we keep these tests and modify them to be with optional upgrades instead?
Since the optional upgrades should still function with upgrade bytes?

Reply:
We don't currently have any actual upgrade to test out. I commented out a working case and added a TODO.